### PR TITLE
Reagan 57 meetingtime enhancements

### PR DESCRIPTION
**Pull request by ReaganTan00**
Modified MeetingTime to be able to accept multiple inputs.
UI changed, similar to Tags.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public static NetWorth parseNetWorth(String netWorth) throws ParseException {
      */
     public static MeetingTime parseMeetingTime(String meetingTime) throws ParseException {
         requireNonNull(meetingTime);
-        String trimmedMeetingTime = meetingTime.trim();
+        String trimmedMeetingTime = meetingTime.replaceAll("\\s", "");
```
Comment (eugenetayyj) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:139]: What is the point of using this replaceAll?
Does this trim the spaces?

File: src/main/java/seedu/address/logic/parser/UpdateCommandParser.java [hunk 2]
```diff
@@ -75,6 +76,23 @@ public UpdateCommand parse(String args) throws ParseException {
         return new UpdateCommand(index, editPersonDescriptor);
     }
 
+    /**
+     * Parses {@code Collection<String> tags} into a {@code Set<Tag>} if {@code tags} is non-empty.
```
Comment (eugenetayyj) [inline src/main/java/seedu/address/logic/parser/UpdateCommandParser.java:80]: might want to change the java docs!

File: src/main/java/seedu/address/model/person/MeetingTime.java [hunk 3]
```diff
@@ -28,9 +29,14 @@ public class MeetingTime {
     public MeetingTime(String meetingTimeString) {
         requireNonNull(meetingTimeString);
         checkArgument(isValidMeetingTime(meetingTimeString), MESSAGE_CONSTRAINTS);
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH:mm");
```
Comment (eugenetayyj) [inline src/main/java/seedu/address/model/person/MeetingTime.java:32]: is there a need to make it caps and uncaps within the pattern? like hour is HH but minute is mm

**Review by eugenetayyj — COMMENTED** [review]
overall alright, LGTM

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (ReaganTan00) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:139]: Yup effectively removes all whitespace
Using trim or strip only removes leading and trailing whitespaces I think, doesnt remove the inbetween ones

File: src/main/java/seedu/address/model/person/MeetingTime.java (hunk 3 above)
Comment (ReaganTan00) [reply src/main/java/seedu/address/model/person/MeetingTime.java:32]: No real purpose to different capitalisation, will just standardise all capitalised if everyone is okay with it

**Pull request merged**